spack.yaml

```diff
@@ -42,10 +47,10 @@ spack:
         - '@1.10.11'
     access-fms:
       require:
-        - '@git.dev_2024.11.0'
+        - '@git.dev_2024.12.0'
     access-generic-tracers:
       require:
-        - '@git.dev_2024.11.0'
```

Comment:
Seems odd to me to specify a version of generic tracers when they're not being used. Not saying it's wrong, just odd.

Reply:
During the upstream Spack meeting, `spack.packages` was described as the the spec if that package is used.

Reply:
So it was! Thanks again.